apps/ui/src/components/ProposalsListItem.vue

```diff
@@ -53,11 +53,9 @@ const handleVoteClick = (choice: Choice) => {
             />
             <div v-else />
           </template>
-          <ProposalVoteBasic
-            v-if="proposal.type === 'basic'"
-            :size="40"
-            @vote="handleVoteClick"
-          />
+          <div v-if="proposal.type === 'basic'" class="py-2">
+            <ProposalVoteBasic :size="40" @vote="handleVoteClick" />
+          </div>
```

Comment:
It is same as this no?

```suggestion
          <ProposalVoteBasic
            v-if="proposal.type === 'basic'"
            class="py-2"
            :size="40"
            @vote="handleVoteClick"
          />
```

Reply:
<img width="195" alt="Untitled 8" src="https://github.com/user-attachments/assets/1c1fc285-30ee-41fe-aeb7-e2cdb2efb116">

Reply:
Correct